Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/GenericCommandExecutionAndEventPublication.php

```diff
@@ -321,6 +321,7 @@ public function theLastCommandShouldHaveThrown(string $shortExceptionName, ?int
                 $this->lastCommandException->getMessage()
             ));
         }
+        $this->lastCommandException = null;
```

Comment:
below we dont do it in `theLastCommandShouldHaveThrownTheWorkspaceRebaseFailedWith` and there is a tihng missing to asser its null after each scenario

Reply:
wait i have a commit ready in 3...2...1